lib/knackhq/client/recursive_symbolize_keys.rb

```diff
@@ -0,0 +1,21 @@
+# Recursively symbolizes keys in hash
```

Comment:
Understand, don't they also usually include the extra functionality or some such extendy thing?